source/features/deep-reblame.tsx

```diff
@@ -41,7 +41,7 @@ const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number, c
 
 	const associatedPR = repository.object.associatedPullRequests.nodes[0];
 
-	if (associatedPR?.number !== prNumber || associatedPR.mergeCommit.oid !== commit) {
+	if (!associatedPR || associatedPR.number !== prNumber || associatedPR.mergeCommit.oid !== commit) {
```

Comment:
I did that ti not have to do ? by all of them

Reply:
Fair, it is more clear 👍